### DISCUSSION TITLE
Get NBA Player averages for our league scoring

### DISCUSSION THREAD

**Discussion by pbcurran** [cwendt94/espn-api#281] [ESPN API] [answered]
Hi love API and really appreciate the work that went into it!
I am tying to get the average league scores for each player in my league. I can see how to get the NBA box score averages for each of the players. I thought I would be able to use this to get our league scores for the players using this method (https://github.com/cwendt94/espn-api/discussions/187) and just multiplying the box scores relevant to our league; however we have bonuses for things like double doubles and triple doubles which are not tracked with these stats so I cant just extrapolate this data from the box scores.

I might be missing an obvious method here (I hope not), but any help would be appreciated. I was thinking something along the lines to get the total score for the players in our league and then just divide by the games played as I can see that this stats is tracked but  Couldn't find a simple way of achieving this. 

Thanks for the help, again I love the work

**Comment by cwendt94** [comment]
Hey @pbcurran, are those special bonuses not added to the players total points? So when you get the box_score and then look at the [box player object](https://github.com/cwendt94/espn-api/wiki/Box-Player-Class-Basketball) the `points` field does not track the bonuses?

**Comment by pbcurran** [comment, marked as answer]
Yeah they are added to the box score, so those scores are accurate for what I need on a per week or game basis. But I wasn't able to see how I can get the total box score of each player for the season so far or the average box score of each player of the season

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by cwendt94** [reply]
Got it, currently you would have to go through box_scores and add up the points per week. However, I don't think it would be too hard adding total points to the [player object]( https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball). I believe that data is part of ESPN's API response.

I will try to look into it tomorrow

**Reply by pbcurran** [reply]
Yeah I thought that might be the case, this is actually what I was trying to avoid haha. Really appreciate the help!

**Reply by cwendt94** [reply]
changes are now in package v0.19.0! [Basketball Player Object](https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball) now has `total_points`, `avg_points`, `projected_total_points`, and `projected_avg_points` fields!

**Reply by pbcurran** [reply]
This is perfect! Thanks so much, you're a legend!
